tests/storage/assert_storage.rs

```diff
@@ -50,8 +52,12 @@ impl AssertionStorage {
     }
 
     pub fn update_with_key(&mut self, cluster: &mut Cluster<ServerCluster>, key: &str) {
-        cluster.must_get(key.as_bytes());
-        let region = cluster.get_region(key.as_bytes());
+        self.update_with_key_byte(cluster, key.as_bytes());
+    }
+
+    pub fn update_with_key_byte(&mut self, cluster: &mut Cluster<ServerCluster>, key: &[u8]) {
+        cluster.must_get(key);
```

Comment:
It sounds all right for me. The issue here is that it's hard to just read tho code and then get the indention of this `must_get()` is to check the leader exists. If you prefer using `must_get()`, please leave a comment to describe that.